qlib/contrib/report/analysis_model/analysis_model_performance.py

```diff
@@ -49,8 +74,17 @@ def _group_return(pred_label: pd.DataFrame = None, reverse: bool = False, N: int
     # Long-Short
     t_df["long-short"] = t_df["Group1"] - t_df["Group%d" % N]
 
-    # Long-Average
-    t_df["long-average"] = t_df["Group1"] - pred_label.groupby(level="datetime")["label"].mean()
+    # Long-benchmark
+    benchmark = kwargs.get("benchmark", "average")
```

Comment:
I think it would be better to have the benchmark as an explicit input parameter, and we can also add type hints.

Reply:
changed as suggested.